CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## Next release
+- Feature: schedule a job to run with delay
```

Comment:
it's a breaking change, can you already document this?
* immediate: true needs to be replaced with delay: 0
* immediate: false can just be omitted